perfect.py

```diff
@@ -0,0 +1,13 @@
+num=input("Enter number : ")
+sum=0
+count=1
+while count<num:
+  if num%count==0:
+    print count,
+    sum=sum+count
+  count=count+1
+
+if sum==num:
+  print "It is a perfect number"
+else:
+  print "It's not a perfect number"
```

Comment:
Again here we can run the while loop only till num/2, and add 1 to final result, which is number itself